docs/src/pages/components/material-icons/SearchIcons.js

```diff
@@ -91,28 +91,41 @@ function selectNode(node) {
 }
 
 let Icons = (props) => {
-  const { icons, classes, handleClickOpen } = props;
+  const { icons, classes, handleOpenClick } = props;
 
-  const handleClick = (event) => {
+  const handleIconClick = (name) => () => {
+    if (Math.random() < 0.1) {
+      window.ga('send', {
+        hitType: 'event',
+        eventCategory: 'material-icons',
+        eventAction: 'click',
+        eventLabel: name,
+      });
+    }
+  };
+
+  const handleLabelClick = (event) => {
     selectNode(event.currentTarget);
   };
 
   return (
     <div>
       {icons.map((icon) => {
         return (
-          <span key={icon.key} className={clsx('markdown-body', classes.icon)}>
+          <span
+            key={icon.key}
+            role="presentation"
```

Comment:
If this is just for the linter then rather disable the rule. This part isn't accessible anyway so let's not interfere even more with aria.